### PR TITLE
CLIP-1748: Update README.md when bumping appVersions

### PR DESCRIPTION
**Pull request by bianchi2**
Adding an additional step to update README.md since this action changes appVersion, and helm-docs uses data from Chart.yaml to update README.md

## Checklist
- [ ] I have added unit tests
- [ ] I have applied the change to all applicable products
- [ ] I have added the change description to the `changelog.md` and `Chart.yaml` files
- [ ] (Atlassian only) I have run the E2E test (if applicable)
- [ ] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

File: .github/workflows/update-lts-tags.yaml [hunk 1]
```diff
@@ -35,6 +40,11 @@ jobs:
           cd src/test/scripts
           python update_versions.py
 
+      - name: Update README files with helm-docs
+        run: |
+          cd src/main/charts
+          helm-docs
```
Comment (nanux) [inline .github/workflows/update-lts-tags.yaml:46]: I think we need to add the template file argument here. We define it for the pre-commit hook as well: https://github.com/atlassian/data-center-helm-charts/blob/main/.pre-commit-config.yaml#LL11C24-L11C24

```
          - --template-files=./README.md.gotmpl
```

**Review by nanux — APPROVED** [review]

File: .github/workflows/update-lts-tags.yaml (hunk 1 above)
Comment (bianchi2) [reply .github/workflows/update-lts-tags.yaml:46]: @nanux yes, but we use the default tpl name, and:

```
  -t, --template-files strings       gotemplate file paths relative to each chart directory from which documentation will be generated (default [README.md.gotmpl])
```

Comment (nanux) [reply .github/workflows/update-lts-tags.yaml:46]: So we can remove it from the `pre-commit-config.yaml`?

Comment (bianchi2) [reply .github/workflows/update-lts-tags.yaml:46]: As long as our tpl file matches the default that helm-docs cli has

**Pull request merged**